frontend/sw360-portlet/src/main/webapp/js/modules/datatables-renderer.js

```diff
@@ -0,0 +1,251 @@
+     *		  	}
+     *		}
+     *  </code>
+     */
+    $.fn.dataTable.render.inputSelect.updateTitle = function(td) {
+        var $select = $(td).find('select');
+        $select.on('change', function() {
+            var $option = $select.find('option[value=' + $select.val() + ']');
+            $select.attr('title', $option.text());
+        });
+    };
+
+    /**
+     * Renders an input text field with the given parameters. This function returns the appropriate render function on call.
+     *
+     * @param {String} name name for the input field
+     * @param {String} clazz class or classes to be added to the select fields. Multiple classes must be space separated
+     * @param {String} placeholder placeholder to be shown inside the input field
+     *
+     * @return {Function} render function for DataTable
+     *
+     * Example usage in column definition: <code>..., renderer: $.fn.dataTable.render.inputText("comment", "comment edit", "Please enter a comment"), ...</code>
+     */
+    $.fn.dataTable.render.inputText = function(name, clazz, placeholder) {
+        return function(value, type) {
+            if(type === 'display') {
+                return createTextInput(name, clazz, placeholder, value)[0].outerHTML;
+            } else if(type === 'type') {
+                return 'string';
+            } else {
+                return value;
+            }
+        }
+    };
+
+    /**
+     * Automatically updates the title attribute of the input field if the value changes.
+     *
+     *  Must be called after a cell has been created.
+     *
+     *  Example datatable configuration:
+     *  <code>
+     *      columnDefs: [{
+     *		  	targets: [1, 6],
+     *		  	createdCell: function(td, cellData, rowData, row, col) {
+     *		  		$.fn.dataTable.render.inputText.updateTitle(td);
+     *		  	}
+     *		}
+     *  </code>
+     */
+    $.fn.dataTable.render.inputText.updateTitle = function(td) {
+        var $text = $(td).find('input');
+        $text.on('change', function() {
+            $text.attr('title', $text.val());
+        });
+    };
+
+    /**
+     * Automatically shows a bigger dialog for entering a value after the original input gained focus.
+     *
+     *  Must be called after a cell has been created.
+     *
+     *  Example datatable configuration:
+     *  <code>
+     *      columnDefs: [{
+     *		  	targets: 2,
+     *		  	createdCell: function(td, cellData, rowData, row, col) {
+     *		  		$.fn.dataTable.render.inputText.useInputDialog("Update create comment");
+     *		  	}
+     *		}
+     *  </code>
+     */
+    $.fn.dataTable.render.inputText.useInputDialog = function(td, dialogTitle) {
+        var $text = $(td).find('input');
+        $text.on('focus', function() {
+            $text.blur(); // unfocus otherwise the dialog will immediately open again after closing
+
+            $('<div><input type="text" style="margin-top: 10px; width: 90%;"/></div>').dialog({
```

Comment:
Yes, you can. Here's their example:
https://craftpip.github.io/jquery-confirm/
![image](https://user-images.githubusercontent.com/16780167/29355613-0c0e5602-8272-11e7-8880-96ab77eb027b.png)